data_rows/WarehouseMeteringHistory.js

```diff
@@ -0,0 +1,49 @@
+import DataRow from "./DataRow";
+import { TYPES } from "./Types";
+import { randomFromArray } from "../helpers";
+
+/**
+ * WAREHOUSE_METERING_HISTORY View
+ *
+ * https://docs.snowflake.net/manuals/sql-reference/account-usage/warehouse_metering_history.html
+ */
+export class WarehouseMeteringHistory extends DataRow {
+  constructor(date) {
+    super();
+    this.setStartAndEnd(date);
+    this.WAREHOUSE_NAME = randomFromArray(["BIG_WH", "SMALL_WH"]);
+    this.CREDITS_USED = Math.random() * 100;
+    if (this.WAREHOUSE_NAME == "SMALL_WH") {
+      this.CREDITS_USED *= 0.3;
+    }
+    this.CREDITS_USED.toFixed(2);
```

Comment:
heads up that .toFixed outputs a string, but the column type here is marked as a float. it probably gets cast somewhere in the data upload, but just making sure you're aware.

Reply:
I didn't know that. yeah, it just "sort of works" and makes the data be 2 points of precision instead of a butt load when it finally gets into the db. I think native type gets kind of lost in writing the csv file and the `Types` stuff we have is just for creating the table columns in snowflake

Reply:
a/ there's a bug I fixed here (needed to assign the result of `toFixed`, it doesn't mutate in place)
b/ now I see the string problem:

```
> const n1 = 0.3252.toFixed(2);
undefined
> const n2 = 0.3652.toFixed(2);
undefined
> n1 + n2
'0.330.37'
```
oops...